genie-common/src/main/java/com/netflix/genie/common/dto/ExecutionEnvironmentDTO.java

```diff
@@ -32,9 +32,10 @@
  * @since 3.0.0
  */
 @Getter
-public abstract class SetupFileAndDependenciesDTO extends CommonDTO {
+public abstract class ExecutionEnvironmentDTO extends CommonDTO {
 
     private static final long serialVersionUID = 2116254045303538065L;
+    protected final Set<String> configs = new HashSet<>();
```

Comment:
I don't think these should be `protected` they should be private and the super classes should use the getters.